src/main.js

```diff
@@ -377,6 +377,81 @@ ${note}
 `
 
 
+const exportToBibTex = data => {
+    var _contents
+
+    // Escape all Tex characters that BibTex requires
+    const badCharReplace = cont => cont.split('')
+        .map(c => {
+            switch (c) {
+                case '#':
+                    return '\\#'
+                case '$':
+                    return '\\$'
+                case '%':
+                    return '\\%'
+                case '&':
+                    return '\\&'
+                case '\\':
+                    return '\\textbackslash{}'
+                case '^':
+                    return '\\textasciicircum{}'
+                case '_':
+                    return '\\_'
+                case '{':
+                    return '\\{'
+                case '}':
+                    return '\\}'
+                case '~':
+                    return '\\textasciitilde{}'
+                default:
+                    return c
+            }
+    }).join('');
+
+    // When missing parameters for author, title, publisher, year
+    var bib_header = '@book{ref' + Math.round(Math.random()*7000).toString() + ':%d' + ',\n'
+
+    if (data.metadata.creator != null && typeof data.metadata.creator !== "undefined") {
+        bib_header += 'author = {%s},\n'.format(badCharReplace(data.metadata.creator))
```

Comment:
Should use template strings instead of `String.prototype.format`. The latter is only needed when working with gettext.